recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/guest-addons.bb

```diff
@@ -11,11 +11,17 @@ LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384
 
 SRC_URI = "\
   file://domu.cfg \
+  file://domc.cfg \
```

Comment:
Is this domc needed here?
If you are keeping it here, then commented lines in domu.cfg looks odd

Reply:
it has different name (so displ_be can be setup properly), IP address and MAC